### DISCUSSION TITLE
Set up Time Delay for Chat Simulation

### DISCUSSION THREAD

**Discussion by RaphMai** [rubenarslan/formr.org#516] [Q&A]
Dear Community,

I would like to add a time delay to my questionnaire so that parts of the questionnaire are displayed after a few seconds.

The reason for this is to simulate a chat interaction.

After a few attempts at troubleshooting and workarounds, I'm back to square one.

So my questions are (again) of a general nature:
How do I successfully embed code?
My previous attempt was to embed JavaScript in HTML. However, I'm now confused as to where to embed the code itself.

I apologise for my naive question and  appreciate any advice.

**Comment by rubenarslan** [comment]
You can simply embed Javascript into the label of a survey item. formr parses Markdown, which means you can also use any HTML. Have you tried this?
